PartTwo.asm

```diff
@@ -0,0 +1,60 @@
+.data
+
+firstLine:	.asciiz "Berkeley Willis, Part Two\nThis takes two numbers from the user and performs three calculations with these numbers and then prints it\n to each terminal as each loads and prints\nPlease enter your first number:"
+promptTwo:	.asciiz "Please enter your second number:"
+sum:		.asciiz "Sum:"
+dif:		.asciiz "\nDifference:"
+pro:		.asciiz "\nProduct:"
+lastMessage:	.asciiz "\nThank you for using my program. Goodbye."
+
+.text
+li $v0, 4, #Prep syscall to print a string
+la $a0, firstLine #pass the arguement for the string
+syscall # Prints the introduction and prompts the user for the first number that the program is ready to recieve
+li $v0, 5	#Prep to read a user's number for the operations
+syscall	#Recieves the user's number and stores it in $v0
+add $t0, $v0, $zero	#Uses the add to zero to store the number from the user
+li $v0, 4	#Preps to print another string
+la $a0, promptTwo	#Loads the string that will print in to register $a0
+syscall	#Prints the second prompt for the user to know to enter a number
+li $v0, 5	#Preps again to receive another number
+syscall	#Recieves the user's number and stores it again in $v0
+add $t1, $v0, $zero	#Uses the add method to store the second number in $$t1
+
+#This section ins the sum part of the program in which it takes the two number and adds them together
+li $v0, 4	#Preps to print a sring
+la $a0, sum	#Loads string I want to print from data
+syscall		#Prints the string
+add $s0, $t0, $t1	#Adds the two numbers together and saves it to an s register
+li $v0, 1	#Preps to print out an integer
+la $a0, ($s0)	#Loads needed register, from the $s0, to $a0
```

Comment:
-2 Line 30: use 'move' or 'add' to shift between registers.  la works but its a roundabout way of doing it because your $s0 is not an address, its just a simple value.